commands/topics_create.js

```diff
@@ -26,7 +26,7 @@ function * createTopic (context, heroku) {
     let addonInfo = yield fetchProvisionedInfo(heroku, addon)
 
     if ((!compaction || addonInfo.shared_cluster) && !retentionTimeMillis) {
-      retentionTimeMillis = addonInfo.limits.minimum_retention_ms
+      retentionTimeMillis = retentionTimeMillis = addonInfo.limits.default_retention_ms || addonInfo.limits.minimum_retention_ms
```

Comment:
Is this double assignment to the same variable a typo? I suggest just

```suggestion
      retentionTimeMillis = addonInfo.limits.default_retention_ms || addonInfo.limits.minimum_retention_ms
```

for better readability.